core/src/test/java/nl/nn/adapterframework/senders/mail/MailSenderTestBase.java

```diff
@@ -80,7 +80,8 @@ public void compare(String file, MimeMessage message) throws Exception {
 			rawResult = rawResult.replace(boundary, "BOUNDARY");
 		}
 
-		TestAssertions.assertEqualsIgnoreCRLF(expected, rawResult);
+		//Make sure there is always a newline, otherwise test assertions will fail
+		TestAssertions.assertEqualsIgnoreCRLF(expected, rawResult.replace("/mixed; boundary", "/mixed; \n\tboundary"));
 	}
```

Comment:
Snap niet waarom hier een newline in zou _kunnen_ zitten

Reply:
Tja het is niet fout als er een newline in zit... de test kan er alleen niet mee overweg